src/Libraries/Microsoft.Extensions.AI.Abstractions/ChatCompletion/ChatOptions.cs

```diff
@@ -168,14 +168,108 @@ public virtual ChatOptions Clone()
 
         if (StopSequences is not null)
         {
-            options.StopSequences = new List<string>(StopSequences);
+            options.StopSequences = [.. StopSequences];
         }
 
         if (Tools is not null)
         {
-            options.Tools = new List<AITool>(Tools);
+            options.Tools = [.. Tools];
         }
 
         return options;
     }
+
+    /// <summary>Merges the options specified by <paramref name="other"/> into this instance.</summary>
```

Comment:
Given that this accepts a nullable, this is an attempt (when applicable not null `other` is provided).

```suggestion
    /// <summary>Attempt to merge the options specified by <paramref name="other"/> into this instance.</summary>
```